execute-generators/src/main/kotlin/de/itemis/mps/gradle/generate/Helper.kt

```diff
@@ -116,7 +119,7 @@ fun generateProject(parsed: GenerateArgs, project: Project): Boolean {
 
     val modelsToGenerate = ftr.get()
 
-    return makeModels(project, modelsToGenerate)
+    return if(modelsToGenerate != null) makeModels(project, modelsToGenerate) else true
```

Comment:
In what situation would modelsToGenerate be null? Should this maybe be reported to the user rather than silently returning `true`?

Reply:
afaics it only happens when the [AsyncPromise](https://github.com/JetBrains/intellij-community/blob/master/platform/util/concurrency/org/jetbrains/concurrency/AsyncPromise.kt) gets explicitly canceled or its value is set to null. Which both doesn't happen. Let's log an error and return false here to fail the build. This way, if some of this reasoning was wrong, we'll find out.

Reply:
should resolve with https://github.com/mbeddr/mps-gradle-plugin/pull/36/commits/ffd675d293197310b6b6bfef315509514881f0ef